src/bitcoin-cli.cpp

```diff
@@ -241,7 +242,20 @@ UniValue CallRPC(const std::string& strMethod, const UniValue& params)
     assert(output_buffer);
     evbuffer_add(output_buffer, strRequest.data(), strRequest.size());
 
-    int r = evhttp_make_request(evcon.get(), req.get(), EVHTTP_REQ_POST, "/");
+    // check if we should use a special wallet endpoint
+    std::string endpoint;
```

Comment:
bug: if no wallet is specified then the endpoint will be "" instead of "/"

result: bitcoin-cli doesn't work without a `-usewallet` parameter:

```
→ bitcoin-cli getinfo
error: no response from server
```

Fix is:

```c++
    std::string endpoint = "/";
```